index.html

```diff
@@ -1,111 +1,579 @@
+          <li>
+            <a href="Catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="Contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <button><a href="link" class="button">Discard</a></button>
+        <button><a href="link" class="button button-blue">Connect</a></button>
+      </div>
+    </header>
+    <main>
+      <section class="statistic">
+        <ul>
+          WITH BINABOX EXPLORE NFT COLLECTION
+          <li>We are tne best way to check the rarity on NTF collection</li>
+          <li>Total item<span class="bold-text">2240+</span></li>
+          <li>Profiles whitelisted<span class="bold-text">1000+</span></li>
+        </ul>
+        <button type="submit">Connect walet</button>
+        <button type="submit">whitelist now</button>
+      </section>
+      <section class="catalog">
+        <div>
+          <h2><span class="folder-icon"></span>Catalog</h2>
+          <div></div>
+          <a href="catalog.html" class="link">See all</a>
+          <button>&#8249;</button>
+          <button>&#8250;</button>
+        </div>
+        <div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="blog">
+        <h2>Our blog</h2>
+        <article class="post">
+          <img src="#" alt="text" />
+          <span class="lable">NTF</span>
+          <span class="folder icon">Tony Nguyen</span>
+          <span class="folder icon">20 Jun 2022</span>
+          <div>
+            <h2>Sonic live</h2>
```

Comment:
It would be better to use the heading 'h3' here, because you have already used 'h2' for headings that indicate sections. To make it easier for you to use headings, try to logically arrange the text in a table of contents like in a book

Reply:
done